modules/datastore/src/Storage/SqliteDatabaseTable.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace Drupal\datastore\Storage;
+
+/**
+ * Database table storage class modified for SQLite.
+ *
+ * Currently only intended for use in unit tests; further testing
+ * needed to determine whether could support a full data catalog.
+ */
+class SqliteDatabaseTable extends DatabaseTable {
+
+  /**
+   * Set the schema using the existing database table.
+   */
+  protected function setSchemaFromTable() {
+    $tableName = $this->getTableName();
+    $fieldsInfo = $this->connection->query("PRAGMA table_info('{$tableName}')")
+      ->fetchAll();
+
+    $schema = $this->buildTableSchema($tableName, $fieldsInfo);
+    $this->setSchema($schema);
+  }
+
+  /**
+   * Get table schema.
+   *
+   * @todo Note that this will breakZ on PostgresSQL
+   */
+  protected function buildTableSchema($tableName, $fieldsInfo) {
+    foreach ($fieldsInfo as $info) {
+      $name = $info->name;
+      $schema['fields'][$name] = $this->translateType(strtolower($info->type), $info);
+      $schema['fields'][$name] = array_filter($schema['fields'][$name]);
+    }
+    return $schema ?? ['fields' => []];
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function translateType(string $type, $info = NULL) {
+    // Clean up things like "int(10) unsigned".
+    $driver = $this->connection->driver() ?? 'mysql';
```

Comment:
Shouldn't the default driver here be "sqlite"?
```suggestion
    $driver = $this->connection->driver() ?? 'sqlite';
```